apps.yml

```diff
@@ -3886,4 +3886,15 @@ apps:
     op: auth0
     url: https://auth.mozilla.auth0.com/samlp/6RMuGl43VwCGOxvFXoDLDXE3U3yhEtT8
     vanity_url:
-    - /securitytraining
\ No newline at end of file
+    - /securitytraining
+- application:
+    AAL: LOW
+    authorized_groups:
+    - everyone
+    authorized_users: []
+    client_id: p2lSKumiDFFnTfopf6zkiF3WB0W359ZK
+    display: true
+    logo: auth0
+    name: Consider.it
+    op: auth0
+    url: https://mozilla.consider.it/
```

Comment:
Is this the correct url? I believe we're using https://mozilla-soti.consider.it

Reply:
Hi @jdow and @gdestuynder, sorry to be annoying, but could we address this pull request? Working on a tight deadline (monday 12/13) unfortunately.

Reply:
I fixed the URL.